### PR TITLE
Added TestMu 2024 in current list

### PR DESCRIPTION
**Pull request by SparshKesari**

### CODE REVIEW

File: _data/current.yml [hunk 1]
```diff
@@ -344,3 +344,10 @@
   url: https://stareast.techwell.com/speak-stareast?utm_source=testingconferences
   twitter: TechWell
   status: CFP is Open until September 10, 2023
+
+- name: Testμ 2024
+  location: Online
+  dates: "20 - 22 August, 2024"
```
Comment (ckenst) [inline _data/current.yml:350]: This isn't the date format we use. Please change

File: _data/current.yml [hunk 2]
```diff
@@ -344,3 +344,10 @@
...
+  url: https://www.lambdatest.com/testmuconf-2024?utm_source=STC&utm_medium=organic&utm_content=testmu_2024
+  twitter: lambdatesting
+  status: Registrations are Open | CFP is Open
```
Comment (ckenst) [inline _data/current.yml:353]: We like to list CFP first and then Registrations second. You can also use html and <a href> tags to provide links to both places in the status.

**Comment by ckenst** [comment]
@SparshKesari thanks for submitting. Please fix the errors and I'll re-review.

**Comment by SparshKesari** [comment]
@ckenst: Please help review and merge

**Review by ckenst — APPROVED** [review]
Looks good.

**Pull request merged**